packages/diffhtml-middleware-verify-state/.babelrc

```diff
@@ -1,6 +1,23 @@
 {
-  "presets": "es2015",
-  "plugins": [
-    "add-module-exports"
-  ]
+  "env": {
+    "build": {
+      "presets": ["es2015"],
+      "plugins": [
+        "add-module-exports",
+        ["module-rewrite", { "replaceFunc": "../../utils/replace-cjs.js" }]
+      ],
+    },
+    "cjs": {
+      "plugins": [
+        "add-module-exports",
+        "transform-es2015-modules-commonjs",
+        ["module-rewrite", { "replaceFunc": "../../utils/replace-cjs.js" }]
+      ]
+    },
+    "es": {
+      "plugins": [
+        ["module-rewrite", { "replaceFunc": "../../utils/replace-es.js" }]
+      ]
+    }
+  }
```

Comment:
Not sure, do you have any suggestions? Our React components repo at work uses a custom tool to abstract away shared config, but that would be too complex to bring into this repo I think.